src/components/Drawer.js

```diff
@@ -131,31 +126,31 @@ const Drawer = React.createClass({
           type='base'
         />
       </nav>
-    ) : <div style={styles.nav} />;
+    ) : <div className={css(styles.nav)} />;
   },
 
   render () {
     const styles = this.styles();
```

Comment:
Cool. I think we can use this technique as well to conditionally generate our styles. Would require more of a shift away from how we're used to doing things.

https://github.com/Khan/aphrodite#conditionally-applying-styles